test/clt-tests/tables-interaction/test-tables-interaction.rec

```diff
@@ -0,0 +1,51 @@
+––– block: ../base/dind/init –––
+––– input –––
+docker pull -q --platform linux/x86_64 manticoresearch/manticore:latest > /dev/null; echo $?
+––– output –––
+0
+––– input –––
+docker pull -q --platform linux/x86_64 manticoresearch/manticore:dev > /dev/null; echo $?
+––– output –––
+0
```

Comment:
Why are we pulling these images? Won't they be pulled automatically in `docker run`?

Reply:
Deleted